design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -157,7 +157,29 @@ input DeleteNegotiableQuotesInput {
     quote_uids: [ID!]! @doc(description: "A List of UIDs obtained from negotiable quote types")
 }
 
+type NegotiableQuoteOperationSuccess {
+    quote_uid: ID!
+}
+
+interface NegotiableQuoteOperationErrorInterface {
+    quote_uid: ID!
+    message: String
+}
+
+type NegotiableQuoteInvalidStateError implements NegotiableQuoteOperationErrorInterface {
+}
+
+type NegotiableQuoteNotFoundError implements NegotiableQuoteOperationErrorInterface {
+}
+
+type NegotiableQuoteUndefinedError implements NegotiableQuoteOperationErrorInterface {
+}
+
+union NegotiableQuoteOperationResult = NegotiableQuoteOperationSuccess | NegotiableQuoteOperationErrorInterface
```

Comment:
This will not work, you cannot have a union type include an interface.  You can still union on all the concrete types and filter on the shared interface in the query though, so the effect is the same.

```suggestion
union NegotiableQuoteOperationResult = NegotiableQuoteOperationSuccess | NegotiableQuoteInvalidStateError | NegotiableQuoteNotFoundError | NegotiableQuoteUndefinedError
```